scripts/create_test

```diff
@@ -257,7 +257,7 @@ OR
         args.no_batch = True
 
     if args.test_id is None:
-        args.test_id = CIME.utils.get_timestamp()
+        args.test_id = "%s.%s"%(CIME.utils.get_timestamp(), random.randrange(0,99,1))
```

Comment:
@billsacks that is probably a little better than uuid since it makes it much more likely that there will be at least on letter and therefore increase the chance that the user will recognize it for what it is.